src/StatusQ/Components/StatusWizardStepper.qml

```diff
@@ -0,0 +1,79 @@
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+Item {
+    id: wizardWrapper
+    width: parent.width
+    height: 56
+    property ListModel stepsModel: ListModel { }
+
+    ListView {
+        id: repeat
+        width: childrenRect.width
+        height: parent.height
+        anchors.horizontalCenter: parent.horizontalCenter
+        orientation: ListView.Horizontal
+        model: stepsModel
+        delegate: Item {
+            id: wrapperItem
+            width: (index === 0) ? descriptionLabel.contentWidth : (wizardWrapper.width / repeat.count)
+            height: 56
+            onXChanged: {
+                //as x changes while delegates are created, direct assignment doesn't work
+                x = (index === (repeat.count-1)) ? (width+32) : (label.width/2) + 8
+            }
+            Rectangle {
+                id: barBorder
+                width: visible ? (parent.width - (label.width/2 + 24)) : 0
+                height: visible ? 8 : 0
+                visible: (index > 0)
+                anchors.verticalCenter: parent.verticalCenter
+                anchors.verticalCenterOffset: -12
+                color: "transparent"
+                border.color: Theme.palette.primaryColor1
+                radius: 4
+                Rectangle {
+                    id: bar
+                    width: loading ? parent.width : 0
+                    Behavior on width { NumberAnimation { duration: loadingTime }}
+                    property bool barLoaded: (width === parent.width)
+                    height: parent.height
+                    radius: 4
+                    color: Theme.palette.primaryColor1
+                }
+            }
+            Item {
+                id: label
+                width: descriptionLabel.contentWidth
+                height: 56
+                anchors.left: (index > 0) ? barBorder.right : parent.left
+                anchors.leftMargin: (index > 0) ? -((width/2) - 24) : 0
+                Rectangle {
+                    width: 32
+                    height: 32
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    radius: width/2
+                    color: stepCompleted ? Theme.palette.primaryColor1 : Theme.palette.indirectColor1
+                    border.color: (stepCompleted && bar.barLoaded) ? "transparent" : Theme.palette.primaryColor1
+                    border.width: 2
+                    StatusBaseText {
+                        anchors.centerIn: parent
+                        text: index+1
+                        font.pixelSize: 17
+                        color: (stepCompleted && bar.barLoaded) ? Theme.palette.indirectColor1 : Theme.palette.primaryColor1
+                    }
+                }
+                StatusBaseText {
+                    id: descriptionLabel
+                    anchors.bottom: parent.bottom
+                    text: description
+                    font.pixelSize: 13
```

Comment:
This will need a `color` to work properly in dark mode

Reply:
done. I used `directColor1` as well here.